spec/system/provider_interface/provider_applications_pagination_spec.rb

```diff
@@ -4,6 +4,8 @@
   include CourseOptionHelpers
   include DfESignInHelpers
 
+  let(:provider) { create(:provider, :with_signed_agreement, code: 'ABC') }
```

Comment:
can we remove this and add @provider = create(:provider, :with_signed_agreement, code: 'ABC') instead please